### PR TITLE
PERFSCALE-2758: Enable reporting mode for not GA versions

### PR DESCRIPTION
**Pull request by rsevilla87**
New PROFILE_TYPE variable, by default both, to select type of indexing

### CODE REVIEW

**Comment by rsevilla87** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-x86-control-plane-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-loaded-upgrade-from-4.12-perfscale-aws-5nodes-loaded-upgrade-412to413to414-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.13-nightly-x86-node-density-heavy-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-node-density-heavy-24nodes

**Comment by rsevilla87** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-24nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-3nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml [hunk 1]
```diff
@@ -79,6 +81,7 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "24"
+      EXTRA_FLAGS: --profile-type=reporting --churn-duration=20m
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml:84]: Don't need churn-duration for the node-density workloads.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-arm.yaml [hunk 2]
```diff
@@ -74,6 +76,7 @@ tests:
       COMPUTE_NODE_REPLICAS: "24"
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      EXTRA_FLAGS: --churn-duration=20m
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-arm.yaml:79]: node-density-heavy .. --profile-type=reporting not churn-duration=20m

File: ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml [hunk 3]
```diff
@@ -18,9 +18,13 @@ ref:
     documentation: |-
       Default is true, which means the workload with delete and recreate a set number of namespaces
   - name: EXTRA_FLAGS
-    default: "--churn-duration=20m"
```
Comment (afcollins) [inline ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml:21]: If you remove this default you do have to add to every consuming step/job. I believe there are some outside of openshift-qe dir, so I would not touch the default here personally.

Comment (rsevilla87) [reply ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml:21]: Added it back

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-arm.yaml (hunk 2 above)
Comment (rsevilla87) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-arm.yaml]: fixed

**Comment by rsevilla87** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-24nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-control-plane-3nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-node-density-heavy-3nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-loaded-upgrade-from-4.12-perfscale-aws-5nodes-loaded-upgrade-412to413to414-periodic

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-control-plane-3nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-node-density-heavy-3nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml [hunk 4]
```diff
@@ -36,6 +36,7 @@ tests:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: candidate
+      CHURN: "false"
```
Comment (rsevilla87) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml:39]: Disabling churning was missing in this job

**Review by jtaleric — COMMENTED** [review]
/lgtm

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml [hunk 5]
```diff
@@ -36,6 +36,7 @@ tests:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: candidate
+      CHURN: "false"
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:39]: What? Seems unrelated to this PR.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml (hunk 4 above)
Comment (afcollins) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml:39]: Would prefer to have unrelated changes in a new PR.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml [hunk 6]
```diff
@@ -54,7 +54,7 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "6"
-      EXTRA_FLAGS: --churn-duration=20m --pod-ready-threshold=13s
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml:57]: we still need churn-duration here, since we are overwriting the default env

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml [hunk 7]
```diff
@@ -54,7 +54,7 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "6"
-      EXTRA_FLAGS: --churn-duration=20m --pod-ready-threshold=13s
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:57]: we still need churn-duration here, since we are overwriting the default env

**Review by afcollins — CHANGES_REQUESTED** [review]
ovnic jobs are missing from this PR.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml (hunk 6 above)
Comment (afcollins) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml:57]: TY!

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml (hunk 5 above)
Comment (rsevilla87) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:39]: Yeah, it's unrelated, new PR?, the fix is pretty obvious

**Comment by krishvoor** [comment]
> ovnic jobs are missing from this PR.

We might need a full-metric collection across 4.15 & 4.16 versions, which is skipped from this change.

**Comment by krishvoor** [comment]
> > ovnic jobs are missing from this PR.
> 
> We might need a full-metric collection across 4.15 & 4.16 versions, which is skipped from this change.

Have had a conversation with Andrew, lgtm!

**Review by afcollins — APPROVED** [review]
lgtm

Thanks @rsevilla87 !

**Review by afcollins — APPROVED** [review]
Excellent. Thanks! @rsevilla87

@jtaleric pls approve

**Review by krishvoor — APPROVED** [review]
/approved
